src/AppBundle/Entity/Role.php

```diff
@@ -65,7 +66,7 @@ class Role extends AbstractPersonalTranslatable implements TranslatableInterface
      * @var Employee
      *
      * @ORM\ManyToOne(targetEntity="AppBundle\Entity\Employee", inversedBy="roles")
-     * @Type("AppBundle\Entity\Employee")
+     * @Type("array")
```

Comment:
@eltrubetskaya API resource MUST return either Object[] or Object in body. This rule should be applied to any resource within REST API